code/modules/mob/living/living.dm

```diff
@@ -59,6 +59,7 @@
 	if(.)
 		if(ranged_ability && prev_client)
 			ranged_ability.remove_mousepointer(prev_client)
+	SEND_SIGNAL(src, COMSIG_LIVING_UPDATE_REVIVABILITY)
```

Comment:
Could you make this signal name more generalized? Something like `COMSIG_LIVING_GHOSTIZED`